tests/test_testbase.py

```diff
@@ -1,20 +1,30 @@
 from sprox.test.base import eq_xml, in_xml, fix_xml
+from sys import version_info
+
+# unordered attrs (python<3.8) are somewhat consistent anyway
+attrs_order_kept = version_info[0] >= 3 and version_info[1] >= 8
 
 def test_fix_xml():
     s = """<form action="" method="post" class="required tableform">
         <div></div>
         </form>"""
-    e = """<form action="" method="post" class="required tableform"><div /></form>"""
-    r =fix_xml(s)
-    assert r == e, (r, s)
+    if attrs_order_kept:
+        e = """<form action="" method="post" class="required tableform"><div /></form>"""
+    else:
+        e = """<form action="" class="required tableform" method="post"><div /></form>"""
+    r = fix_xml(s)
```

Comment:
Instead of `if:else:`  given that the purpose of `fix_xml` is mostly to get rid of empty spaces etc... Why not just test for `r.contans("><div /></form>")`?